<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ordinateur</class>
 <widget class="QWidget" name="Ordinateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>624</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="Assimilation">
        <property name="title">
         <string>Assimilation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Assimiler toutes les</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="DeltaT">
           <property name="suffix">
            <string> s</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.017000000000000</double>
           </property>
           <property name="maximum">
            <double>60.000000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Pendant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="TotalT">
           <property name="suffix">
            <string> s</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>86400.000000000000000</double>
           </property>
           <property name="value">
            <double>60.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Donnees">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>140</height>
         </size>
        </property>
        <property name="title">
         <string>Données</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="Grandeur">
           <property name="maxVisibleItems">
            <number>11</number>
           </property>
           <item>
            <property name="text">
             <string>Aucune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>t (en s)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>V (en mL)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Θ (en °C)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Θ (en °K)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>pH</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[H30+] (en mol.L⁻¹)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[HO-] (en mol.L⁻¹)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>σ (en S.cm⁻¹)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>T (en %)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Abscisse">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Définir en abscisse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OrdonneeG">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Définir en ordonnée (gauche)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OrdonneeD">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Définir en ordonnée (droite)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MinMax">
        <property name="text">
         <string>Définir valeurs des axes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Assimiler">
        <property name="text">
         <string>[A]ssimiler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Reset">
        <property name="text">
         <string>Effacer le graphique</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Export">
        <property name="text">
         <string>Exporter le graphique</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Graphique">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
